geometry_msgs/Vector3 pos_est
int32 vehicle_index
